### PR TITLE
Enabling access to new collection

### PR DESCRIPTION
**Pull request by tamas-molnar**
* don't know why we need to do this every time...

### CODE REVIEW

**Comment by Nesze** [comment]
+1

**Comment by tosan88** [comment]
Doesn't need to be updated puppet template file as well? https://github.com/Financial-Times/nativerw/blob/master/puppet/ft-nativerw/templates/config.json.erb

**Comment by tamas-molnar** [comment]
You're right Sandor, added it.

**Comment by tosan88** [comment]
cheers!

File: config.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
 	  "port" : "8080"
 	},
 	"dbName" : "native-store",
-  	"collections" : ["methode","wordpress"]
+  	"collections" : ["methode","wordpress","methode-metadata"]
```
Comment (martingartonft) [inline config.json]: Wouldn't this be better called "v1-metadata"? it's not specific to methode.

Comment (tamas-molnar) [inline config.json]: Yeah, you're right, will change.

**Pull request merged**